tree/ntuple/v7/inc/ROOT/RNTupleReadOptions.hxx

```diff
@@ -44,14 +44,22 @@ private:
    EClusterCache fClusterCache = EClusterCache::kDefault;
    unsigned int fClusterBunchSize = 1;
    EImplicitMT fUseImplicitMT = EImplicitMT::kDefault;
+   /// If true, the RNTupleReader will track metrics straight from its construction, as
+   /// if calling `RNTupleReader::EnableMetrics()` before having created the object.
+   bool fEnableMetrics = false;
 
 public:
    EClusterCache GetClusterCache() const { return fClusterCache; }
    void SetClusterCache(EClusterCache val) { fClusterCache = val; }
+
    unsigned int GetClusterBunchSize() const { return fClusterBunchSize; }
    void SetClusterBunchSize(unsigned int val) { fClusterBunchSize = val; }
+
    EImplicitMT GetUseImplicitMT() const { return fUseImplicitMT; }
    void SetUseImplicitMT(EImplicitMT val) { fUseImplicitMT = val; }
+
+   bool AreMetricsEnabled() const { return fEnableMetrics; }
```

Comment:
We typically use the (unwritten) convention of boolean functions starting with `Has...`
```suggestion
   bool HasMetricsEnabled() const { return fEnableMetrics; }
```